views/errordisplay.ejs

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>SIGN UP</title>
+  <link rel="stylesheet" type="text/css" href="/css/style.css" />
+</head>
+
+<body>
+  <form action="/signup" method="POST" class="forms">
+  <h4>SIGN UP</h4>
+      <label for="username"><b>Username</b></label>
+      <input type="text" placeholder="Enter your Name" id="username" name="username">
+      <br>
+      <label for="password"><b>Password</b></label>
+      <input type="password" placeholder="Enter your password" id="password" name="password">
+      <br>
+      <label for="email"><b>Email ID</b></label>
+      <input type="email" placeholder="Enter your Email ID" id="email" name="email">
+      <br>
+      <button type="submit" class="submit">SUBMIT</button>
+  </form>
+
+  <h5>PLEASE ENTER VALID CREDENTIALS</h5>
+  <% if(typeof alert != 'undefined') {%>
+    <% alert.forEach(function(error){ %>
+        <div class="errormsg" role="alert">
+            <%= error.msg %>
+          </div>
+        <% }) %>
+  <% } %>
+
+  <% if (alerts === 'Valid') { %>
```

Comment:
the condition check for "Valid"  and you still display an error message.  The code is confusing.